cpp/QuickSort.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+ 
+using namespace std;
+ 
+void quick_sort(int[],int,int);
+int partition(int[],int,int);
+ 
+int main()
+{
+    int a[50],n,i;
+    cout<<"How many elements?";
+    cin>>n;
+    cout<<"\nEnter array elements:";
+    
+    for(i=0;i<n;i++)
```

Comment:
Please follow a style guide while submitting C++ code. I suggest going through https://google.github.io/styleguide/cppguide.html for this